torch/_dynamo/eval_frame.py

```diff
@@ -113,7 +116,27 @@ def _maybe_set_eval_frame(callback: DynamoCallback):
         )
         return callback
     else:
-        return set_eval_frame(callback)
+        assert state in ["enter", "exit"]
+        if torch._dynamo.compiled_autograd.in_compiled_autograd_region:
+            # NOTE: Compiled Autograd graph cannot be executed in eager mode and must be Dynamo traced through
```

Comment:
we should reword this, the graph can be executed in eager mode, but in this case you don't want the warmup flag to apply to the torch.compile on the compiled autograd graph